Makefile

```
@@ -189,24 +189,27 @@ image-builder-version: .github/env
 image: GOOS := linux # Overriding GOOS value for docker image build
 image: .hack-operator-image .hack-prometheus-config-reloader-image .hack-admission-webhook-image
 
-.hack-operator-image: Dockerfile
+.PHONY: .hack-operator-image
 # Create empty target file, for the sole purpose of recording when this target
 # was last executed via the last-modification timestamp on the file. See
 # https://www.gnu.org/software/make/manual/make.html#Empty-Targets
+.hack-operator-image:
```

(nit) let's rename the target to something more friendly and remove the comment since it doesn't apply anymore.
```suggestion
.PHONY: operator-image
operator-image:
```

---

I did make the name change initially, but then now we will also have to change the `gitignore`. So I left it as is. Now worries! I will make the change.